packages/widget_toolkit_otp/lib/widget_toolkit_otp.dart

```diff
@@ -0,0 +1,24 @@
+library widget_toolkit_otp;
+
+// Expose package specific enums which we would otherwise need to reimplement
+export 'package:pinput/pinput.dart'
```

Comment:
I believe in this case we can afford reimplementing those enums so that we don't need to export third-party package specific classes.